node/src/service.rs

```diff
@@ -621,7 +542,7 @@ pub fn new_dev(
 				client: client.clone(),
 				pool: transaction_pool.pool().clone(),
 				commands_stream,
-				select_chain,
+				select_chain: maybe_select_chain.expect("constructed when in dev service"),
```

Comment:
Is this handling a should-be-impossible case, or telling the user that they should invoke moonbeam differently? I wonder if this could be expressed more clearly...

Reply:
fixed in fcab602